tools/upgrade_versions.py

```diff
@@ -114,13 +114,17 @@ def update(self, user_name, user_email):
 
     def _bump_release(self, pkg, log=None, user_name=None, user_email=None):
```

Comment:
Does it make sense to accept a list here? I'd accept a str instead and create the log list with the str as the list first element. This would remove the need of making changelog variable above a list, which does not make sense IMHO